src/welcome/getting-started.js

```diff
@@ -0,0 +1,164 @@
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'Introduction to Wireframes', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Use the Design Library', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+		{
+			title: __( 'How to Use the Column Blocks', i18n ),
+			subtitle: __( 'Lorem ipsum dolor sit amet consectetur. Molestie in arcu est feugiat pulvinar sit.', i18n ),
+			src: 'https://www.youtube.com/embed/MaXOZ-1UIvs',
+			url: '#',
+		},
+	],
+	headerTitle: __( 'Learn the essentials', i18n ),
+	headerSubtitle: __( 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Consequat tempus sollicitudin rhoncus pretium tellus.', i18n ),
+}
+
+const Section = props => {
+	const Card = ( {
+		title, subtitle, link, icon = '', src = '',
+	} ) => {
+		return <div className="s-card">
+			{ props.label === 'general'
+				? <div className="s-icon-wrapper"> { icon } </div>
+				: <div className="s-video-wrapper">
+					<iframe className="s-video" src={ src } title={ title } allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture;" allowFullScreen></iframe>
+				</div>
+			}
+			<h3 className="s-card-title"> { title } </h3>
+			<p className="s-card-subtitle"> { subtitle } </p>
+			<div className="s-card-link">
+				<a href={ link }>
+					{ props.label === 'general' ? <SVGArrowUpRightIcon />
+						: __( 'Watch Video', i18n )
+					}
+				</a>
+			</div>
+		</div>
+	}
+
+	return <Fragment>
+		{ props.label !== 'general' &&
+			<div className="s-card-header">
+				<h2> { props.headerTitle } </h2>
+				<p> { props.headerSubtitle } </p>
+			</div>
+		}
+		<div className={ `s-card-container s-card-${ props.label }` }>
+			{ props.content.map( ( item, key ) => {
+				return <Card { ...item } key={ key } />
+			} ) }
+		</div>
+	</Fragment>
+}
+
+export const GettingStarted = () => {
+	return <>
+		<div className="s-body">
+
+			<div className="s-getting-started__centered">
+				<h2 className="title"> { __( 'Unleash the Full Potential of the WordPress Block Editor by Turning It into a Page Builder', i18n ) } </h2>
+				<p className="subtitle">{ __( 'Learn the essentials in just a few minutes by watching this video. Scroll down to see more quick tutorials.', i18n ) } </p>
+
+				<div className="s-video-wrapper s-getting-started-video">
+					<iframe className="s-video" src="https://www.youtube.com/embed/MaXOZ-1UIvs" title="Getting Started" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture;" allowFullScreen></iframe>
```

Comment:
Please make sure that all readable strings are translatable:

`title={ __( 'Getting Started', i18n ) }`